UITests/baseFile.html

```diff
@@ -0,0 +1,9 @@
+<html>
+    <head>
+        <link rel="icon" href="image.png"></link>
```

Comment:
Hm, not sure I like doing this here. I'd prefer each test having the minimal page necessary instead of trying to create on single shared file that tries to do too much. I guess this is fine for now, though -- let's just be careful to prevent this from growing out of control.